src/components/PoweredBy/PoweredBy.tsx

```diff
@@ -32,7 +32,9 @@ export const PoweredBy = ({ styles }: PoweredByProps) => {
     `${process.env.NEXT_PUBLIC_SITE_URL}/${currentPath}`,
   );
 
+  //Todo: logic to review
   const isRoot = result === '/' || result === '';
+  const isThemePage = result === '/memecoins';
```

Comment:
Not sure, but the way I have set up such paths in "urls.ts". We may want to add this "/memecoins", too.

Reply:
not sure to understand.